include/coap2/coap_io.h

```diff
@@ -51,6 +61,14 @@ typedef struct coap_socket_t {
   coap_fd_t fd;
 #endif /* WITH_LWIP */
   coap_socket_flags_t flags;
+  struct coap_session_t *session; /* Used for the epoll logic if a session.
```

Comment:
s/if/of/?

Reply:
Actually `if` is correct.  However have changed the comment to `Used by the epoll logic if a session.`

Reply:
This does not seem to be a complete sentence.

Reply:
How about `Used by the epoll logic for an active session`?
- and for endpoint `Used by the epoll logic for a listening endpoint` ?

Reply:
Sounds good.

Reply:
Fixed.